src/go/cmd/http-relay-server/server/server.go

```diff
@@ -129,40 +129,55 @@ func extractBackendNameAndPath(r *http.Request) (backendName string, path string
 	return
 }
 
+type responseChunk struct {
+    Body []byte
```

Comment:
please indent

Reply:
You were too quick with your review :) Now all files are auto-formatted.